src/test/ui/nll/issue-58299.rs

```diff
@@ -0,0 +1,27 @@
+struct A<'a>(&'a ());
```

Comment:
This should use `#![feature(nll)]` (or a `-Zborrowck=mir` revision) otherwise the error from the old region checker is shown.

Reply:
Fixed.